src/main/java/sirius/biz/storage/Storage.java

```diff
@@ -424,7 +427,8 @@ public void updateFile(@Nonnull StoredObject file,
             virtualObjectCache.remove(object.getObjectKey());
 
             // Delete all (now outdated) versions
-            oma.select(VirtualObjectVersion.class).eq(VirtualObjectVersion.VIRTUAL_OBJECT, object).delete();
```

Comment:
könnte man auch in den afterdelete handler von VirtualObjectVersion packen - dann tuts immer net nur meistens